index.js

```diff
@@ -605,6 +605,11 @@
            other.unicode === this.unicode;
   }
 
+  //  RegExp$prototype$lte :: RegExp ~> RegExp -> Boolean
+  function RegExp$prototype$lte(other) {
+    return lte(toString(this), toString(other));
+  }
```

Comment:
> I don't see the immediate utility.

A binary search tree needs to compare values during insertion and lookup. It seems reasonable to expect to be able to store RegExp objects in such a tree.